<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Q_WordrecitingProc</class>
 <widget class="QWidget" name="Q_WordrecitingProc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>千词斩 Version 1.0</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>30</y>
     <width>300</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>峰峰的日常</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_0">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>94</y>
     <width>180</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>较难词汇：0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>180</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>生疏词汇：100</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>186</y>
     <width>180</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>掌握词汇：0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Word">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>130</y>
     <width>300</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>prescription</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_Yes">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>120</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>我认识</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_No">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>120</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>我不认识</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_Abandon">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>350</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>弃疗</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_Next">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>350</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>下一题</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Meaning">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>190</y>
     <width>300</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>n. 药方，对策，开处方</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_No">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>260</y>
     <width>260</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;稍后将继续安排这个单词的学习！&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Yes">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>260</y>
     <width>260</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;今天不再安排学习！&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_Cancel">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>撤销</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_Kill">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>290</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>斩</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_kill">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>290</y>
     <width>260</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;您已经斩掉了这个单词，以后不再安排学习！&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
